### PR TITLE
Iterator Helpers are now in stage 3

### PR DESCRIPTION
**Pull request by JamieMagee**
https://github.com/tc39/proposal-iterator-helpers/pull/251

### CODE REVIEW

File: testers.json [hunk 1]
```diff
@@ -3565,217 +3565,217 @@
...
       "spec": "https://github.com/tc39/proposal-iterator-helpers",
       "code": "const array = [1, 2, 3].values().toArray();\nreturn Array.isArray(array) && array.join() === '1,2,3';"
     },
-    "Stage 2›Iterator Helpers›Iterator.prototype[@@toStringTag]": {
+    "Stage 3›Iterator Helpers›Iterator.prototype[@@toStringTag]": {
       "spec": "https://github.com/tc39/proposal-iterator-helpers",
       "code": "return Iterator.prototype[Symbol.toStringTag] === 'Iterator';"
     },
-    "Stage 2›Iterator Helpers›instanceof AsyncIterator": {
+    "Stage 3›Iterator Helpers›instanceof AsyncIterator": {
```
Comment (ljharb) [inline testers.json:3636]: async iterator helpers are still stage 2.

File: testers.json [hunk 2]
```diff
@@ -3565,217 +3565,217 @@
...
-    "Stage 2›Iterator Helpers›AsyncIterator.prototype[@@toStringTag]": {
+    "Stage 3›Iterator Helpers›AsyncIterator.prototype[@@toStringTag]": {
       "spec": "https://github.com/tc39/proposal-iterator-helpers",
       "code": "return AsyncIterator.prototype[Symbol.toStringTag] === 'AsyncIterator';"
+    },
+    "Stage 2›Generator function.sent Meta Property": {
+      "spec": "https://github.com/tc39/proposal-function.sent",
+      "code": "var result;\nfunction* generator() {\n  result = function.sent;\n}\nvar iter = generator();\niter.next('tromple');\nreturn result === 'tromple';"
+    },
+    "Stage 2›Class and Property Decorators›class decorators": {
```
Comment (ljharb) [inline testers.json:3712]: decorators are stage 3 now

File: testers.json [hunk 3]
```diff
@@ -3565,217 +3565,217 @@
...
+    },
+    "Stage 2›Set methods›Set.prototype.union": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "var set = new Set([1, 2]).union(new Set([2, 3]));\nreturn set.size === 3\n  && set.has(1)\n  && set.has(2)\n  && set.has(3);"
+    },
+    "Stage 2›Set methods›Set.prototype.difference": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "var set = new Set([1, 2, 3]).difference(new Set([3, 4]));\nreturn set.size === 2\n  && set.has(1)\n  && set.has(2);"
+    },
+    "Stage 2›Set methods›Set.prototype.symmetricDifference": {
```
Comment (ljharb) [inline testers.json:3744]: set methods are stage 3 now

File: testers.json [hunk 4]
```diff
@@ -3565,217 +3565,217 @@
...
+    },
+    "Stage 2›Set methods›Set.prototype.isSubsetOf": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "return new Set([1, 2, 3]).isSubsetOf([5, 4, 3, 2, 1]);"
+    },
+    "Stage 2›Set methods›Set.prototype.isSupersetOf": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "return new Set([5, 4, 3, 2, 1]).isSupersetOf([1, 2, 3]);"
+    },
+    "Stage 2›ArrayBuffer.prototype.transfer›ArrayBuffer.prototype.transfer()": {
```
Comment (ljharb) [inline testers.json:3760]: transfer and transferToFixedLength and .disposed are stage 3 now (realloc is gone)

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by ljharb** [comment]
@JamieMagee do you no longer have interest in completing this PR?

**Comment by JamieMagee** [comment]
Sorry, I don't have capacity (or the knowledge) to make the changes.

**Pull request closed without merging**